src/index.js

```diff
@@ -0,0 +1,41 @@
+import './style.css';
+
+import Awesome from './modules/classAwesome.js';
+
+const enter = document.querySelector('.enter');
+
+const awesome = new Awesome();
+
+const clear = document.querySelector('.clear-btn');
+
+const addListenerOnFocus = (chores) => {
+  chores.addEventListener('focusout', (e) => {
+    const item = e.target.value;
+    awesome.updateInput(item, e.path[0].defaultValue);
+    awesome.local();
+  });
+};
+
+// Main load
+document.addEventListener('DOMContentLoaded', () => {
+  awesome.returnInfo();
+  const chores = document.querySelectorAll('.label-input');
+  chores.forEach((chore) => addListenerOnFocus(chore));
+});
+
+enter.addEventListener('click', (e) => {
+  e.preventDefault();
+  const chore = document.getElementById('myInput').value;
+  if (!chore) {
+    alert('missing information');
```

Comment:
Please avoid using `window.alert()` . It's impossible to style and displays differently on different browsers. It would be best to customize the error message using css.